solr/contrib/ltr/example/solrconfig.xml

```diff
@@ -857,7 +857,10 @@
     </requestHandler>
 
  <!-- Query parser used to rerank top docs with a provided model -->
-  <queryParser name="ltr" class="org.apache.solr.ltr.ranking.LTRQParserPlugin" />
+  <queryParser name="ltr" class="org.apache.solr.ltr.ranking.LTRQParserPlugin" >
+  <int name="LTRMaxThreads">10</int> <!-- Maximum threads to use for all queries -->
+  <int name="LTRMaxQueryThreads">10</int> <!-- Maximum threads to use for a single query-->
```

Comment:
README should be updated with new params, thoroughly explaining what they do, and when to use them.